src/sentry/snuba/metrics/extraction.py

```diff
@@ -1527,6 +1527,29 @@ def _get_satisfactory_threshold_and_metric(project: Project) -> tuple[int, str]:
     return threshold, metric_field
 
 
+def _escape_wildcard(value: str) -> str:
+    """
+    Escapes all characters in the wildcard which are considered as meta characters in the glob
+    implementation in Relay, which can be found at: https://docs.rs/globset/latest/globset/#syntax.
+
+    The goal of this function is to only preserve the `*` character as it is the only character that Sentry's
+    product offers to users to perform wildcard matching.
+    """
+    i, n = 0, len(value)
+    escaped = ""
+
+    while i < n:
+        c = value[i]
+        i = i + 1
+
+        if c in "[]{}?":
```

Comment:
I have checked in the Relay glob implementation and these are the only characters that can be escaped if we want the exact match.